services/common/src/components/tailings/AssociatedDams.tsx

```diff
@@ -113,12 +114,12 @@ const AssociatedDams: FC<AssociatedDamsProps> = (props) => {
         renderActionsColumn({ actions }),
     ];
 
-    const mostRecentUpdatedDate = moment(
+    const mostRecentUpdatedDate = tsf.dams.length ? moment(
         Math.max.apply(
             null,
             tsf.dams.map((dam) => moment(dam.update_timestamp))
         )
-    ).format("DD-MM-YYYY H:mm");
+    ).format("DD-MM-YYYY H:mm") : EMPTY_FIELD;
```

Comment:
just about forgot- fixed the "Last Updated: Invalid Date" to show N/A instead of Invalid Date